backend/portal/views.py

```diff
@@ -99,7 +99,7 @@ def get_queryset(self):
             )
         )
 
-    @action(detail=False, methods=["post"])
+    @action(detail=False, methods=["post"])  # WHY IS THIS POST
```

Comment:
`POST` because the devices send us a `id_hash` which is a hash based on device id. This is because we want to associate votes based on this rather than pennkey to keep users anonymous. For this, if the user voted we don't given them the ability to vote again (based on if `id_hash` already voted)

Reply:
wait wat, but i don't see any posting going on here. am i blind? it looks like its just updating stuff

Reply:
L108 ` id_hash = request.data["id_hash"]`

Reply:
but there's no up updating to our models going on? like can't we just use a get request for this

Reply:
oh wait nvm. https://stackoverflow.com/questions/19637459/rest-api-using-post-instead-of-get. so we can send through the body im assuming?

Reply:
Yea right. I think generally I like sending things via POST body as opposed to query params if it isn't literally for a non trivial query